deploy/README.md

```diff
@@ -0,0 +1,9 @@
+# Deployment
+
+The Compliance application uses the [UBI8/ruby27](https://catalog.redhat.com/software/containers/ubi8/ruby-27/5f521b90acb94fdd904e98d7?gti-tabs=unauthenticated) base image. 
+
+You can create a new image using:
+
+```
+podman build -t compliance-backend:latest .
```

Comment:
```
podman build -t compliance-backend:latest -f deploy/Dockerfile .
```
since the docker file is not in the root. I guess for Clowder (for builds within app-interface) it would have to be in the root.

Reply:
Let's make the production Dockerfile the root :) https://github.com/RedHatInsights/compliance-backend/pull/836

Reply:
Sure, both is fine with me. :+1:

Reply:
Done, moved current `entrypoint.sh` to `devel.entrypoint.sh` and renamed it's usage in `devel.Dockerfile`, moved everything from `deploy/` to root dir.